code/__DEFINES/bloodcult.dm

```diff
@@ -0,0 +1,2 @@
+var/global/blood_target = null

```

Comment:
Could you point me to a good example of this? Cult variables are kind of scattered to the 4 winds and I'm not even sure where I'd concentrate them or how I'd get the alert code to recognize them (thats why I took the lazy round with globals).